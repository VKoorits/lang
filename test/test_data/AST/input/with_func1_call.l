i=0
while i < 10:
	if i%2:
		print(i*i)
	else:
		c = i*i - 4*i + 1
		pritn(c)
return(0)

