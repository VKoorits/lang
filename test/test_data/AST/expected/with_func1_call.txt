i 0 = EXPR_0 T_BODY_1 while CALL_2 NULL
EXPR_0 :i 10 < NULL
T_BODY_1 :EXPR_3 T_BODY_4 F_BODY_5 if NULL
EXPR_3 :i 2 % NULL
T_BODY_4 :CALL_6 NULL
CALL_6 :ARGS_7 print NULL
ARGS_7 :i i * NULL
F_BODY_5 :c i i * 4 i * 1 + - = CALL_6 NULL
CALL_6 :ARGS_7 pritn NULL
ARGS_7 :c NULL
CALL_2 :ARGS_3 return NULL
ARGS_3 :0 NULL
